code/Hieu_L/python/classes_lab.py

```diff
@@ -0,0 +1,54 @@
+import random
+class bank_account:
+    def __init__(self, name, balance, account_number):
+        self.name = name
+        self.balance = balance
+        self.account_number = random.randint(0,9)
+    def __repr__(self):
+        return print(f"account number: {self.account_number} \n name: {self.name} \n account number: {self.account_number}")
+
+    def deposit(self):
+        amount = input('enter deposit amount: ')
+        self.balance += amount
+        return self.balance
+    
+    def withdrawal(self):
+        amount = input('enter withdrawal amount: ')
+        self.balance -= amount
+        return self.balance
+
+class rectangle:
+    def __init__(self, length, width):
+        self.length = length
+        self.width = width
+    
+    def area(self):
+        area = self.length * self.width 
+        return area
+    
+    def perimeter(self):
+        perimeter = (2 * self.length) + (2 * self.width)
+        return perimeter
+
+    def __repr__(self):
+        return print(f"rectangle dimensions: \n length: {self.length} \n width: {self.width} \n area:")
```

Comment:
This should either be a return or a print. Not both. Although, it will still work like this. It will return None (the result of the print function) but print because it is still a print function.